src/data/faq.json

```diff
@@ -201,7 +201,7 @@
                         "anchor": "eu_dcc_remove",
                         "active": true,
                         "textblock": [
-                            "Using this step-by-step guide, you can delete certificates from the Corona-Warn-App. Please note that the first three steps apply to both iOS and Android, but only one of the last two steps needs to be followed, depending on which operating system matches yours. Vaccination certificates, test certificates and certificates of recovery are all called 'Certificates' in the following, as the process of removing them is identical.<br>After you have performed the following steps, the certificate is moved to the recycle bin for the time being. There, it will then be automatically deleted after 30 days, unless you restore it from there or permanently delete it (the recycle bin feature is available starting with version 2.13).",
+                            "Using this step-by-step guide, you can delete certificates from the Corona-Warn-App. Please note that the first three steps apply to both iOS and Android, but only one of the last two steps needs to be followed, depending on which operating system matches yours. Vaccination certificates, test certificates and certificates of recovery are all called 'Certificates' in the following, as the process of removing them is identical.<br>After you have performed the following steps, the certificate is moved to the recycle bin for the time being. There, it will then be automatically deleted after 30 days, unless you restore it from there or permanently delete it. (the recycle bin feature is available starting with version 2.13.)",

```

Comment:
@Ein-Tim 
If you add a full stop after "it", then you need to capitalize "The" in  "The recycle bin ..." which follows.

Reply:
@MikeMcC399 
Thank you!
Fixed with [`09b3dbb`](https://github.com/corona-warn-app/cwa-website/pull/2262/commits/09b3dbbc41229a0308372ae1161536859b5371ab).